pkg/service/catalog/api/types.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+type CatalogService struct {
+	// Target is a reference to a service
+	Target kapi.ObjectReference
+	// Type is a published service's claim type
+	Type string
+	// Description is a user-defined description of the published service
+	Description string
```

Comment:
> Why would we do this as an annotation here? That doesn't make sense to me...

For generic display purposes consistent with all other types.  @liggitt, that example is on an embedded type that doesn't have an `ObjectMeta`.  The only top level examples I know of are the `ThirdPartyResource` stuff and `ProjectRequest`.